at91lib/boards/tfrog-rev5/board.h

```diff
@@ -221,6 +221,8 @@
 #define PINS_EEPROM_TWCK   {0x00000010, AT91C_BASE_PIOA, AT91C_ID_PIOA, PIO_OUTPUT_1, PIO_PULLUP | PIO_OPENDRAIN}
 #define PINS_EEPROM        {0x00000018, AT91C_BASE_PIOA, AT91C_ID_PIOA, PIO_PERIPH_A, PIO_PULLUP | PIO_OPENDRAIN}
 
+#define PIN_MOTOR_POWER {AT91C_PIO_PB26, AT91C_BASE_PIOA, AT91C_ID_PIOA, PIO_OUTPUT_1, PIO_DEFAULT}
```

Comment:
This name looks modified version of the board specific.
`PIN_EXTERNAL_POWER_SWITCH`?